git-test/src/components/Login.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react';
+import { TextField, Button } from '@mui/material';
+
+function Login() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [savedData, setSavedData] = useState<{ email: string, password: string } | null>(null);
+
+  const handleSubmit = (event: { preventDefault: () => void; }) => {
+    event.preventDefault();
+    setSavedData({ email, password });
+  };
+
+  const handleShowData = () => {
+    if (savedData) {
+      alert(`Email: ${savedData.email}, Password: ${savedData.password}`);
+    }
+  };
+
+  return (
+    <form onSubmit={handleSubmit} style={{
```

Comment:
don't use inline styles, you can style with MUI like this: const StyledForm = styled('form')({});